gpii/node_modules/settingsHandlers/test/WebSocketsSettingsHandlerTests.js

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Many more tests required before we can merge this in - including
i) tests which actually verify that the WebSockets endpoint does something - including testing correct handling of multiple simultaneous clients which connect/disconnect in different sequences
ii) tests to check that repeated sequences of get/set settings have the expected effect